apps/frontend/src/dashboard/Dashboard.tsx

```diff
@@ -34,6 +34,12 @@ const Dashboard: React.FC<Record<string, never>> = () => {
     filters: new Set(),
   });
   usePrefetchCalendars();
+  usePrefetchResearchOutputs({
```

Comment:
I actually extended `usePrefetchTeams` and some others to have a default config with these values so it doesn't pollute the components that just want some simple prefetching as much. Should use that here as well.